spec/player_turn_spec.rb

```diff
@@ -1,20 +1,20 @@
 require "spec_helper"
-require_relative "../lib/player_input.rb"
+require_relative "../lib/player_turn.rb"
 require_relative "../lib/coordinates.rb"
 require_relative "../lib/grid.rb"
 require_relative "../lib/printer.rb"
 
-RSpec.describe PlayerInput do
+RSpec.describe PlayerTurn do
   describe "#get_valid_coordinate" do
      it "repeatedly asks for valid coordinate, returns the value when valid" do
       invalid_format = StringIO.new
```

Comment:
I think this should just be `stdin` or `input_stream`. It's hard to imagine its role when it is named that way.

Reply:
I think for now I'll go back to `PlayerInput`.